src/Microsoft.TestPlatform.CrossPlatEngine/AttachmentsProcessing/DataCollectorAttachmentProcessorAppDomain.cs

```diff
@@ -157,7 +157,9 @@ public async Task<ICollection<AttachmentSet>> ProcessAttachmentSetsAsync(XmlElem
         cancellationToken.Register(() => _wrapper.CancelProcessAttachment());
         _processAttachmentSetsLogger = logger;
         _progressReporter = progressReporter;
+#pragma warning disable CS8603 // Possible null reference return.
```

Comment:
Can you add an explanation of what's wrong here?

Reply:
Deserialization of the result might be null. Not sure why I did not see that before.